tools/cli/go-whisk-cli/commands/action.go

```diff
@@ -505,16 +505,20 @@ func webActionAnnotations(
         var action *whisk.Action
         var err error
 
-        if annotations != nil || !fetchAnnotations {
-            annotations = webActionAnnotationMethod(annotations)
-        } else {
+        if fetchAnnotations {
             if action, _, err = client.Actions.Get(entityName); err != nil {
-                return nil, actionGetError(entityName, err)
+                whiskErr, isWhiskError := err.(*whisk.WskError)
+
+                if (isWhiskError && whiskErr.ExitCode != whisk.EXIT_CODE_NOT_FOUND) || !isWhiskError {
+                    return nil, actionGetError(entityName, err)
+                }
             } else {
-                annotations = webActionAnnotationMethod(action.Annotations)
+                annotations = append(annotations, action.Annotations...)
```

Comment:
If `annotations` and `action.Annotations` each define the same annotation, I think the returned `annotations` slice will have that annotation defined twice, with the existing annotation value appearing ordinally after the updated value.  If so, will downstream consumers select the first or last annotation value?